keyboards/helix/rev3_4rows/keymaps/via/keymap.c

```diff
@@ -0,0 +1,311 @@
+/* Copyright 2020 yushakobo
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#include QMK_KEYBOARD_H
+
+#ifdef RGBLIGHT_ENABLE
+//Following line allows macro to read current RGB settings
+extern rgblight_config_t rgblight_config;
+#endif
+
+// Defines names for use in layer keycodes and the keymap
+enum layer_names {
+  _QWERTY = 0,
+  _LOWER,
+  _RAISE,
+  _ADJUST
+};
+
+// Defines the keycodes used by our macros in process_record_user
+enum custom_keycodes {
+  EISU = SAFE_RANGE,
+  KANA,
+  ADJUST,
+  RGBRST
+};
+
+#define LOWER MO(_LOWER)
+#define RAISE MO(_RAISE)
```

Comment:
RAISE and LOWER should use `FN_MO13` and `FN_MO23`, as these produce the tri-layer stuff.

https://github.com/qmk/qmk_firmware/blob/ed2ba645c05fe3a0b8dcf5ac8be4ec2426c4917c/quantum/via.c#L163-L182

Eg, you would want: 
```suggestion
#define LOWER FN_MO13
#define RAISE FN_MO23
```